person.rb

```diff
@@ -0,0 +1,62 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+class Person < Nameable
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    super()
+    @id = rand(1..1_000_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
As part of the project requirements, kindly make sure the add_rental method in the person class should create a new rental (like Rental.new) with the book and date as parameters. The person itself should also be sent as a parameter to create the new rental to help achieve the 'has-many' association.
Kindly check [this link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more information on that. :wink: